app/controllers/admin/applications_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::ApplicationsController < ApplicationController
+  def show
+    @application = Application.find(params[:id])
+    @pets = @application.get_pets
```

Comment:
It's okay to create @pets here but also keep in mind that you can call @application.get_pets in the view, too, and that's not a violation of MVC

Reply:
We want to encapsulate any AR queries beyond CRUD in model methods. Then in the view, we stay away from doing any AR queries and instead just call those model methods that we made